### ISSUE TITLE
CVXOPT GLPK params not passing through

### ISSUE THREAD

**Issue by cmale** [cvxgrp/cvxpy#251]
When using the GLPK and GLPK_MI solvers with cvxpy, setting solver parameters such as
```
prob.solve(solver=cvx.GLPK_MI, verbose=False, tmlim=60)
```
do not get passed through to the solver. The simpliest example is setting verbose=False on any problem, you will see that all outputs are still printed to stdout. 

To fix this I went into the specific solver interface file: cvxpy/problems/solvers/glpk_mi_intf.py and set the glpk options directly instead of using the generic cvx solvers options

```python
if verbose:
    #cvxopt.solvers.options["msg_lev"] = "GLP_MSG_ON"
    cvxopt.glpk.options["msg_lev"] = "GLP_MSG_ON"
else:
    #cvxopt.solvers.options["msg_lev"] = "GLP_MSG_OFF"
    cvxopt.glpk.options["msg_lev"] = "GLP_MSG_OFF"
```
This is a workaround, but there should be a better fix to use the generic cvxopt solver options. 

For reference I am using cvxpy 0.3.4 and cvxopt 1.1.8

**Comment by SteveDiamond** [comment]
Thanks for catching this! I haven't used GLPK very much so I'm not familiar with the solver options. Could you figure out the right way to pass in the solver options and submit a pull request?

**Comment by cmale** [comment]
I created a pull request. Found that the GLPK ILP solver is not included in the CVXOPT LP solvers, so params cannot be set through cvxopt.solvers.options, and instead need to be explicitly set with cvxopt.glpk.options. This works for GLPK_MI.

According to CVXOPT documentation, options for regular GLPK LPs should work using CVXOPT solvers options. However, with some quick tests with verbose I did not find this to be true. Could be a CVXOPT issue.

http://cvxopt.org/userguide/coneprog.html#algorithm-parameters

>The control parameters listed in the GLPK documentation are set to their default values and can be customized by making an entry in solvers.options. The keys in the dictionary are strings with the name of the GLPK parameter. For example, the command

>from cvxopt import solvers
solvers.options['msg_lev'] = 'GLP_MSG_OFF'
turns off the screen output subsequent calls lp with the 'glpk' option.
